modules/page-memory/src/integrationTest/java/org/apache/ignite/internal/pagememory/tree/AbstractBplusTreePageMemoryTest.java

```diff
@@ -3265,15 +3206,6 @@ protected static void println(@Nullable String msg) {
         System.out.println(msg);
```

Comment:
Holy cow, it's time to change it to logging.

Reply:
Not in this PR

Reply:
Ok